frontend/src/pages/Faq/Faq.jsx

```diff
@@ -37,172 +39,131 @@ export function Faq(props) {
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                Suspendisse malesuada lacus ex, sit amet blandit leo lobortis
-                eget.
-              </Typography>
-            </AccordionDetails>
-          </Accordion>
-          <Accordion
-            className={dark ? `${style["accord-dark"]}` : `${classes.accord}`}
-            expanded={expanded === "panel4"}
-            onChange={handleChange("panel4")}
-          >
-            <AccordionSummary
-              style={{ color: "white" }}
-              expandIcon={
-                <ExpandMoreIcon style={{ color: "white", fontSize: "27px" }} />
-              }
-              aria-controls="panel1a-content"
-              id="panel1a-header"
-            >
-              <h3 className={style["faq-question"]}>
-                <i className="fa fa-question-circle" aria-hidden="true"></i>
-                &nbsp; &nbsp;Is there any custom pricing system?
-              </h3>
-            </AccordionSummary>
-            <AccordionDetails>
-              <Typography style={{ color: "white" }}>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                Suspendisse malesuada lacus ex, sit amet blandit leo lobortis
-                eget.
-              </Typography>
-            </AccordionDetails>
-          </Accordion>
-          <Accordion
-            className={dark ? `${style["accord-dark"]}` : `${classes.accord}`}
-            expanded={expanded === "panel5"}
-            onChange={handleChange("panel5")}
+          <img
+            src="./images/FAQ2.png"
+            className={style["faq-image"]}
+            alt="faq-icon"
+          />
+        </div>
+        <div className={style["faq"]}>
+          <div
+            className={
+              dark
+                ? `${style["faq-block"]} ${style["faq-block-dark"]}`
+                : `${style["faq-block"]} ${style["faq-block-light"]}`
+            }
           >
-            <AccordionSummary
-              style={{ color: "white" }}
-              expandIcon={
-                <ExpandMoreIcon style={{ color: "white", fontSize: "27px" }} />
-              }
-              aria-controls="panel1a-content"
-              id="panel1a-header"
-            >
-              <h3 className={style["faq-question"]}>
-                <i className="fa fa-question-circle" aria-hidden="true"></i>
-                &nbsp; &nbsp;How to change my password?
-              </h3>
-            </AccordionSummary>
-            <AccordionDetails>
-              <Typography style={{ color: "white" }}>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                Suspendisse malesuada lacus ex, sit amet blandit leo lobortis
-                eget.
+            {isFetching ? (
+              <Typography variant="h6" align="center" style={{color:"white"}}>
+                Please wait while we fetching FAQs
```

Comment:
Please user loader instead of this, while fetching the data

Reply:
@Kajol-Kumari  made the required changes. Review it